packages/expo-crypto/src/Crypto.ts

```diff
@@ -142,3 +142,22 @@ export async function digestStringAsync(
 
   return await ExpoCrypto.digestStringAsync(algorithm, data, options);
 }
+
+/**
+ * The `getRandomValues()` method of `Crypto` fills a provided `TypedArray` with cryptographically secure random values.
+ *
+ * @param typedArray A `TypedArray` to fill with cryptographically secure random values.
```

Comment:
I'd rather just link like this:

A [`TypedArray`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray) to fill with cryptographically secure random values.

instead of listing all as I think we'd need to do that everywhere we accept or return a typed array.

wdyt?